### ISSUE TITLE
Migrate `user_profile` endpoints to new API structure

### ISSUE THREAD

**Issue by anth-volk** [PolicyEngine/policyengine-api#1989]

**Comment by mikesmit** [comment]
picking this up and basing the style on https://github.com/PolicyEngine/policyengine-api/pull/2038 and the modifications Anthony has been making.

**Comment by mikesmit** [comment]
I have done a version of this here: https://github.com/PolicyEngine/policyengine-api/tree/1989_user_profile_to_blueprint but I need @anth-volk 's changes from his PR to actually finalize it (specifically the exception response binding).

**Comment by mikesmit** [comment]
picking this back up now to rebase on anthony's change. This should fix a major security issue.

**Comment by mikesmit** [comment]
ok rebase fixed the failing unit tests (for error handling). In the interests of plugging the immediate security issue I'm submitting that PR now (without breaking out service tests)

**Comment by mikesmit** [comment]
opened pull request: https://github.com/PolicyEngine/policyengine-api/pull/2067

**Comment by mikesmit** [comment]
oops forgot to add the versioning stuff. doing that now..

**Comment by mikesmit** [comment]
Yes. I corrected the link in the commit, but it does not auto-update the message (thanks for fixing)

Root cause of test failure is a difference between the mysql syntax (which does not allow " around column names) and the desktop test database: SQLite which does.

**Comment by mikesmit** [comment]
Merging this change this morning. Will check to make sure it does't break anything.
